lib/mix/phoenix/context.ex

```diff
@@ -46,6 +51,7 @@ defmodule Mix.Phoenix.Context do
       web_module: web_module(),
       basename: basename,
       file: file,
+      test_dir: test_dir,
```

Comment:
Do you need the test_dir because the test_file doesn't have the notifier suffix? What if you add the notifier suffix before you call `Context.new`?

Reply:
> Do you need the test_dir because the test_file doesn't have the notifier suffix? 

@josevalim the idea was to only get the context test dir, without the notifier. I changed to infer this from `test_file` now. WDYT? (I didn't get your suggestion though :thinking:)